app/views/studies/_form.html.erb

```diff
@@ -119,16 +119,14 @@
 
     ClassicEditor
       .create( document.querySelector( '#study_description' ),
-          {
-              toolbar: ["heading", '|',  "bold", "italic", "link", "bulletedList", "numberedList", "blockQuote", '|', "undo", "redo"]
-          }
+          fullCKEditorConfig
       )
       .then( function(editor) {
-          console.log(Array.from( editor.ui.componentFactory.names() ));
+          console.log('Editor inserted');
       } )
```

Comment:
Like we remove debug logging in Ingest Pipeline, it's also worth removing debug logging here.  If needed, this can be debugged in production via DevTools.